src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -1522,6 +1587,17 @@ public long getTruncateRpcTimeout()
         return DatabaseDescriptor.getTruncateRpcTimeout(MILLISECONDS);
     }
 
+    public void setTransactionTimeout(long value)
+    {
+        DatabaseDescriptor.setTransactionTimeout(value);
+        logger.info("set transaction timeout to {} ms", value);
+    }
+
+    public long getTransactionTimeout()
+    {
+        return DatabaseDescriptor.getTransactionTimeout(MILLISECONDS);
+    }
```

Comment:
we should use `String` when talking to users as that's the API they know; we did this for most of the other configs as well.